app/styles/app.scss

```diff
@@ -725,74 +725,79 @@ hr {
             }
         }
-        & > b {
+        .subject-container {
+            background-color: $brand-primary;
             display: inline-block;
-            padding: 5px 12px ;
-            position: relative;
-            border-radius: 4px;
-            &%pseudo {
-                content: '';
-                display: block;
-                position: absolute;
-                width: 0;
-                height: 0;
-                left: 100%;
-                top: 50%;
-                margin-top: -5rem;
-                border: {
-                    style: solid;
-                    width: 5rem 0 5rem 3rem;
-                    color: transparent transparent transparent $brand-primary;
+            transition: background-color 0.1s linear;
+            margin-left: -10px;
+            padding-left: 10px;
+            & > b {
+                display: inline-block;
+                padding: 5px 15px ;
+                position: relative;
+                border-radius: 4px;
+                &%pseudo {
+                    content: '';
+                    position: absolute;
+                    width: 0;
+                    height: 0;
+                    left: 100%;
+                    top: 0;
+                    border: {
+                        style: solid;
+                        width: 15px 0 15px 12px;
+                        color: transparent transparent transparent $brand-primary;
+                    }
+                }
+                &:before {
+                    @extend %pseudo;
+                    margin-left: 1px;
+                    border-left-color: white;
+                    z-index: 1;
+                }
+                &:after {
+                    @extend %pseudo;
+                    transition: border-color 0.1s linear;
+                    z-index: 2;
                 }
-            }
-            &:before {
-                @extend %pseudo;
-                margin-left: 1px;
-                border-left-color: white;
-                z-index: 1;
-            }
-            &:after {
-                @extend %pseudo;
-                transition: border-color 0.1s linear;
-                z-index: 2;
-            }
-            &:first-child {
-                padding-left: 0;
             }
             &:last-of-type {
-                padding-right: 2rem;
-                &:before, &:after {
+                & > b::before, & > b::after {
                     display: none;
                 }
-            }
-            & + b {
-                margin-left: 2rem;
-            }
-        }
-        & > .fa {
-            color: white;
-            cursor: pointer;
-            transition: color 0.1s linear;
-            &:hover {
-                color: lighten(red, 40%);
-            }
-            &:active {
-                color: lighten(red, 20%);
+                & > b > .cancel-subject {
+                    &:before {
+                        font: {
```

Comment:
For the most part, we use font awesome's predefined classes for this kind of thing when possible. It's more readable in the markup, and has the additional benefit of not redefining something that is already there. Eg, `<i class="fa fa-close"></i>`

I might be missing something- is there a particular advantage to redefining the CSS here?